docs/advanced-configuration.md

```diff
@@ -195,6 +195,30 @@ logsCollection:
 
 Use https://regex101.com/ to find a golang regex that works for your format and specify it in the config file for the config option `firstEntryRegex`.
 
+
+### Collect journald events
+
+Splunk OpenTelemetry Collector for Kubernetes can collect journald events from kubernetes environment.
+Process journald events by configuring `logsCollection.journald` section in values.yaml.
+
+```yaml
+logsCollection:
+  journald:
+    enabled: true
+    directory: /run/log/journal
+    defaultPriority: info
+    # List of service units to collect and configuration for each. To collect all journald services logs, leave units blank.
+    units:
+      - name: kubelet
+        priority: info
+      - name: docker
+        priority: info
+      - name: containerd
+        priority: info
```

Comment:
Why not enable all systemd sources by default, and provide `units` configuration as a commented out option? It would be nice to have only one journald receiver by default, not 3.

Reply:
updated